src/components/requests/drawing.vue

```diff
@@ -9,27 +15,38 @@
       channel.postRequest('DrawTools.StartDrawingRequest', data);"></code-component>
     </div>
     <div id="StopDrawingRequest">
-      <button class="btn btn-primary exampleready" @click="stopDrawing">DrawTools.StopDrawingRequest - cancel current drawing</button>
+      To stop current drawing progress send <code>StopDrawingRequest</code>.
+      <button class="btn btn-primary exampleready" @click="stopDrawing">DrawTools.StopDrawingRequest</button>
```

Comment:
The current example have replaced ` <button class="btn btn-primary exampleready">` with `<RunExampleButton>`. Use it instead.

Reply:
Check!